android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/AutostartReceiver.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2015 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.physical_web.physicalweb;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+
+/**
+ * This receiver starts the UriBeaconDiscoveryService
+ */
+
+public class AutostartReceiver extends BroadcastReceiver {
+  public void onReceive(Context context, Intent intent) {
+    SharedPreferences sharedPreferences = context.getSharedPreferences(context.getString(R.string.physical_web_preference_file_name), Context.MODE_PRIVATE);
```

Comment:
Lines shouldn't be longer than 100 characters. 

https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.4-column-limit